server/index.js

```diff
@@ -12,7 +14,14 @@ app.use(bodyParser.urlencoded({ extended: false }));
 
 // respond with "hello world" when a GET request is made to the homepage
 app.get('/api/songs/:songId', (req, res) => {
-  res.send('hello world');
+  res.send(data.songs);
 });
 
 app.listen(port, () => console.log(`Your server has connected and is listening on port: ${port}!!`));
+
+  // used to populate the db
+  // for (let i = 0; i < data.songs.length; i++) {
```

Comment:
any particular reason you're doing this on the server side rather than the db side? i did the population on the database file but am not sure what the ideal is

Reply:
I just did it because I could control when to do the upload of data by sending a get request through postman, instead of reloading the db or server. I wasn't sure when the db server code would run things, if it was just synchronously whenever I loaded or what and so I just opted for this.